apps/settings/style/lists.css

```diff
@@ -161,7 +162,8 @@ ul li > label.pack-checkbox > .color-preview:before {
   height: 2rem;
   width: 2rem;
   content: "";
-  margin: -1.1rem 0 0 -3rem;
+  margin-top: -1.1rem;
+  -moz-margin-start: -3rem;
```

Comment:
Are you sure we’re inheriting a `margin: 0` here? Otherwise, it should be added above the `margin-top` rule.